### ISSUE TITLE
script blank screen

### ISSUE THREAD

**Issue by Cakei244** [EchoControl/Gunkolle#17]
**Desktop:**
What graphics card are you using?
GeForce GT 740M

**Config File:**
Post your Config File
[Do Not Modify]
Busy=0
[Variables]
SortieInterval=1000
MaxRandomWaitS=3000
MinRandomWaitS=3000
MinRandomWait=0
MaxRandomWait=0
WINID=ahk_class Qt5QWindowIcon
PauseHr=9
PauseMn=00
ResumeHr=12
ResumemN=00
Doll1=AR15
Doll2=M4SOPMODII
Doll3=ERROR
Doll4=ERROR
WeaponType=AssaultRifle
Enhancement=0
ProductionTdoll=0
ProductionEquipment=0
FriendCollector=0
BatteryCollector=0
AutoBattleResend=0
WorldSwitcher=5_4_friendly
LastXS=-32000
LastYS=-32000
[GPU]
AMD=0
#For AMD or others GPU's change value 0 to 1. For NVidia value must be 0

#combatsims can be of values 1,2, or 3
CombatSimsData=2

LastXS=860
LastYS=775
**Describe the bug**
The script is just a blank screen, dont show nothing more
I try the original one, and is show a menu and all, but with the Echo is just a blank screen
**Screenshots**
Add screenshots with your android player and where the script has stopped.
![Nox_vvTCTQN7MF](https://user-images.githubusercontent.com/56614349/66878356-0da03900-ef7f-11e9-8288-c050473093c5.png)
![AutoHotkey_6BTaxfn4f4](https://user-images.githubusercontent.com/56614349/66878366-13961a00-ef7f-11e9-8d51-79e6f8304c4e.png)
![explorer_USyx6pqvi8](https://user-images.githubusercontent.com/56614349/66878370-16910a80-ef7f-11e9-84dd-6621bc8ff833.png)

**Comment by EchoControl** [comment]
```
LastXS=860
LastYS=775 

```
Reset those to 0 and make sure to run it with admin rights

**Comment by Cakei244** [comment]
Thanks now is showing info, i want to use ldplayer emulator, 
![as8cxGLMmK](https://user-images.githubusercontent.com/56614349/66957620-30d4f200-f02c-11e9-9936-d019b1ddf066.png)

but when i click start it say that Noxplayer not found it
![LCJOaNY9yN](https://user-images.githubusercontent.com/56614349/66957646-3cc0b400-f02c-11e9-8f27-d0bf0a3dc007.png)

and this is when i select debug
![3itoPFJKgP](https://user-images.githubusercontent.com/56614349/66957670-42b69500-f02c-11e9-88a3-8efb554c9401.png)

i modifie this line of the config already
WINID=ahk_class Qt5QWindowIcon
with 
WINID=ahk_class LDPlayerMainFrame
or
ahk_exe dnplayer.exe
i select one, with both is the same error.

**Comment by EchoControl** [comment]
Not sure about other emulators as I only tested it on NOX

**Comment by yocool13** [comment]
From what I've tried it definitely only works with Nox. Although it doesn't work anymore at all with the new October Client Update.

**Comment by EchoControl** [comment]
Yup, I'll look at it starting today, hopefully I'll get it back to work soon
